chassis/configuration/configuration/reporter/reporting.conf.xml

```diff
@@ -226,7 +226,7 @@
     </bean>
 
     <alias name="defaultReportingService" alias="reportingService"/>
-    <bean id="defaultReportingService" class="com.griddynamics.jagger.reporting.ReportingService">
+    <bean id="defaultReportingService" class="com.griddynamics.jagger.reporting.ReportingService" scope="prototype">
```

Comment:
Why do we need prototype scope for the reportigService bean? 

As I can see we use the service in 2 ways:
In Master.java -- to render the report after test execution
In JaggerLauncher.java -- to run standalone report generation